virttest/qemu_devices/qcontainer.py

```diff
@@ -1064,12 +1064,14 @@ def pflash_handler(firmware_name, cmd):
                 path = storage.get_image_filename_filesytem(img_params,
                                                             current_data_dir)
                 img_info = json.loads(qemu_storage.QemuImg(img_params,
-                                                           data_dir.DATA_DIR,
+                                                           current_data_dir,
                                                            path).info(True, "json"))
                 img_name = os.path.basename(img_info.get("filename"))
 
                 pflash_vars_name = "%s_%s_VARS.fd" % (self.vmname, img_name)
                 # Force the image file to be saved under "images" directory
+                # FIXME: Put the pflash_var file and the image file in the same
```

Comment:
Hello @pevogam @luckyh , what's your opinion about it?
Without `images_base_dir`, I prefer to put all images created by Avocado-vt under `DATA_DIR/images`, but now images can be provided if `images_base_dir` is defined by users, so let them in the same directory also make senses with me.

Reply:
+1, thanks.

Reply:
Yes, makes sense, +1.